yapf/__init__.py

```diff
@@ -226,6 +215,20 @@ def main(argv):
       verbose=args.verbose)
   return 1 if changed and (args.diff or args.quiet) else 0
 
+def print_help(args):
```

Comment:
It looks like the contents of this function are over-indented. You should use two spaces.

Reply:
RIght you are. Updated.